File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target)
    {
        int l = 0;
        int r = numbers.Length - 1;
        while (l<r)
        {
            int curSum = numbers[l] + numbers[r];
            if (curSum > target)
            {
                r--;
            }
            else if (curSum < target)
            {
                l++;
            }
            else
            {
                return [l+1, r+1];
            }
        }
        
        return Array.Empty<int>();
    }
}

File: Data Structures & Algorithms/append-characters-to-string-to-make-subsequence/submission-0.cpp
class Solution
{
public:
    int appendCharacters(string s, string t)
    {
        int s1 = 0;
        int t1 = 0;
        while (s1 < s.size() && t1 < t.size())
        {
            if (s[s1] == t[t1])
            {
                t1++;
            }
            s1++;
        }

        return t.size() - t1;
    }
};